app/domain/queries/find_content.rb

```diff
@@ -26,6 +27,8 @@ def initialize(filter)
     @to = filter.fetch(:to)
     @organisation_id = filter.fetch(:organisation_id)
     @document_type = filter.fetch(:document_type)
+    @page = filter[:page] || 1
+    @page_size = filter[:page_size] || 100
```

Comment:
Shall we extract a constant with the default page size, so we don't have magic numbers?